#The depth for the spot
float64 depth

#The latitude for the spot
float64 lat_spot

#The longtitude for the spot
float64 long_spot
